lib/jekyll-seo-tag/json_ld.rb

```diff
@@ -51,6 +51,8 @@ def json_image
         hash = image.dup
         hash["url"]   = hash.delete("path")
         hash["@type"] = "imageObject"
+        keepers = ["@type", "height", "width", "url"]
+        hash.keep_if { |k, _| keepers.include?(k) }
```

Comment:
Here is my attempt to handle the extra tags that exist in the image hash and not using them for the output.